skiplang/prelude/ts/src/sk_types.ts

```diff
@@ -950,3 +950,16 @@ export function cloneIfProxy<T>(v: T): T {
   }
   return v;
 }
+
+export function errorObjectAsError(e: ErrorObject): Error {
```

Comment:
What is `ErrorObject` and why not have it subclass `Error` directly?

Reply:
Indeed, I need to check